src/scripts/components/story-list/story-list.styl

```diff
@@ -1,9 +1,14 @@
 @require '../../../variables.styl'
 
 .storyList
-  display: grid
-  padding: 20px
+  overflow-y: scroll
```

Comment:
`.storyList` and `.storyListGrid` have `overflow-y: scroll`. Are you sure we need both divs/classes?